src/app/docs/sample/sample.component.html

```diff
@@ -7,7 +7,7 @@
     Sample descriptionn.
   </sky-docs-demo-page-summary>
 
-  <sky-docs-demo>
+  <sky-docs-demo supportsTheming="true">
```

Comment:
This evaluates to "truthy"; but it should probably be `[supportsTheming]="true"`.